modules/internal/ChapelStandard.chpl

```diff
@@ -24,25 +24,20 @@ module ChapelStandard {
   use startInitCommDiags;  // Internal, but uses standard/CommDiagnostics
 
   // Internal modules.
-  use CPtr;
   use CString;
   use Bytes;
   use String;
   use OwnedObject;
   use SharedObject;
-  use ChapelDebugPrint;
   use ChapelEnv;
   use ChapelBase;
-  use MemConsistency;
   use Atomics;
   use NetworkAtomics;
   use NetworkAtomicTypes;
   use AtomicsCommon;
   use ChapelLocks;
   use ChapelIteratorSupport;
   use ChapelThreads;
-  use ChapelThreadsInternal;
-  use ChapelTasksInternal;
```

Comment:
👍 For getting rid of these. 

I also think we could remove ChapelThreads

Reply:
Hmm maybe not, there's a few uses of it in studies/madness/aniruddha/madchap/, but those could be `here.maxTaskPar`.

Reply:
I considered that, but thought I found references to its contents still...

Reply:
Yeah, there are a few uses for studies/madness/aniruddha/madchap/* -- work for a future PR